data/yam/agama/auto/lib/scripts.libsonnet

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I find really unlikely that we use a pre script also as post script, wouldn't make more sent to separate them in different files, so the user of this feature can see clearly the catalog for each.